submissions/cynthiaNwakaeme/domManipulation/newsletter.js

```diff
@@ -0,0 +1,22 @@
+let sendForm = document.getElementById("requiredField"),
+    newsletter = document.getElementById("newsletter"),
+    errorMessage = document.querySelector(".errorMessage");
+
+// To validate the Newsletter
+sendForm.addEventListener("submit", require = (event) => {
+    const emailRegex = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,10})+$/;
+
+    if (newsletter.value === "") {
+        errorMessage.innerHTML = "Please enter your email";
+        event.preventDefault();
+        return false;
+    } else
```

Comment:
The `else if` statement should be on the same line @Cynthia-Berry

Reply:
@ToluAkin  Good Day ma,
I have effected the changes.


Thank you for observing that.
Kindly review my PR.